components/02-molecules/social-links/social-links.twig

```diff
@@ -0,0 +1,26 @@
+{% set social_links__baseclass = 'social-links' %}
+
+<div {{ bem(social_links__baseclass) }}>
+  {# list <ul> #}
+  {# social-links__list #}
+  <ul {{ bem('list', [], social_links__baseclass) }}>
+    {# For each loop #}
+    {% for icon in icons %}
+      {# list-items <li> #}
+      {# social-links__item #}
+      <li {{ bem('item', [], social_links__baseclass) }}>
+        {# link <a> #}
+        <a href="{{ icon.url }}" {{ bem('link', [], social_links__baseclass) }}>
+          {# Visually hidden service name (Facebook) #}
+          <span class="visually-hidden">{{ icon.name }}</span>
+          {# Visually shown icon #}
+          {% include '@atoms/images/icons/_icon.twig' with {
+            icon__name: icon.icon,
+            icon__decorative: true,
+            icon__blockname: social_links__baseclass,
+          } %}
+        </a>
+      </li>
+    {% endfor %}
+  </ul>
+</div>
```

Comment:
Need a new line at the end of the file. We'll get your editor setup to do this automatically in the future.

Reply:
line space added